sql/sql_tablespace.cc

```diff
@@ -247,6 +247,13 @@ bool lock_tablespace_names(THD *thd, Names... names) {
     return true;
   }
 
+  // Acquire Percona's LOCK TABLES FOR BACKUP lock
+  if (thd->backup_tables_lock.abort_if_acquired() ||
+      thd->backup_tables_lock.acquire_protection(
+          thd, MDL_TRANSACTION, thd->variables.lock_wait_timeout)) {
+    return true;
+  }
+
```

Comment:
can you also please add a test that shows LTFB is taken and DMLS on tablespaces happen after LTFB is taken?

Reply:
done

Reply:
I dont see it

Reply:
check this in patch INSERT into t1 values